ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/PivotSelectorFromSafeBlock.java

```diff
@@ -131,15 +131,15 @@ public CompletableFuture<Void> prepareRetry() {
 
   private FastSyncState selectLastSafeBlockAsPivot(final Hash safeHash) {
     LOG.debug("Returning safe block hash {} as pivot", safeHash);
-    return new FastSyncState(safeHash);
+    return new FastSyncState(safeHash, true);
   }
 
   private FastSyncState selectFallbackBlockAsPivot(final Hash fallbackBlockHash) {
     LOG.debug(
         "Safe block not changed in the last {} min, using a previous head block {} as fallback",
         UNCHANGED_PIVOT_BLOCK_FALLBACK_INTERVAL / 60,
         fallbackBlockHash);
-    return new FastSyncState(fallbackBlockHash);
+    return new FastSyncState(fallbackBlockHash, true);
```

Comment:
Not so sure about this, as the fallback is not a safe block, but the head block from the fcu. I guess the source is still trusted, but the source does not have a lot of confidence in that hash.